packages/typespec-ts/src/index.ts

```diff
@@ -407,6 +408,15 @@ export async function $onEmit(context: EmitContext) {
         buildPackageFile(model, modularPackageInfo)
       );
       commonBuilders.push(buildTsConfig);
+
+      if (option.generateTest && isAzureFlavor) {
```

Comment:
Could you add a TODO comment for multi-client cases? https://github.com/Azure/autorest.typescript/issues/3048